app/javascript/src/components/menus/editable_collection_item_menu.js

```diff
@@ -35,13 +34,14 @@ class EditableCollectionItemMenu extends ActivatedMenu {
       this.selection(event, ui.item);
     });
 
-    let $target = this._config.$target;
+    let $target = this.config.$target;
```

Comment:
Although the original alluded to config being private while accepting JS limitation of keeping it public, with the underscore name prefix now removed we've also removed that indication it should be private. Is that intentional? The inherited ActivatedMenu class has done the full job of making it actually private (with modern hash naming), so I'm guessing this should be similar.

Again, not going to hold approval of PR but might be worth making a note to revisit when the dust settles.

Reply:
Ah no, I looked further.  This is in the constructor so its using the passed in config directly.  This menu never actually assigns the passed in `config` to an internal `#config` var as it's not needed.